sdk/python/feast/infra/offline_stores/snowflake_source.py

```diff
@@ -26,7 +26,6 @@ def __init__(
         name: Optional[str] = None,
         timestamp_field: Optional[str] = "",
         database: Optional[str] = None,
-        warehouse: Optional[str] = None,
```

Comment:
This should be kept in, and just raise a deprecation warning when specified. Otherwise it'll break users pretty bad

Reply:
Moving to draft -- will add deprecation flag

Reply:
added back